File: docs/misc/codesnippet/VisualBasic/walkthrough-extending-managed-vspackages-by-using-automation_2.vb
    Private Sub MenuItemCallback(ByVal sender As Object, ByVal e As EventArgs)
        Dim dte As DTE
        dte = CType(GetService(GetType(DTE)), DTE)
        Dim myString As String = "Name is " & dte.Name + vbCrLf & "Version is " & dte.Version

        Dim windows As Windows = dte.Windows
        Dim window As Window = CType(windows.Item(EnvDTE.Constants.vsWindowKindOutput), Window)
        window.Visible = True

        Dim outputWindow As OutputWindow = CType(window.Object, OutputWindow)
        Dim outputWindowPane As OutputWindowPane = outputWindow.OutputWindowPanes.Add("Test")
        outputWindowPane.Activate()
        outputWindowPane.OutputString(myString)
    End Sub

File: docs/misc/codesnippet/CSharp/walkthrough-extending-managed-vspackages-by-using-automation_2.cs
        private void MenuItemCallback(object sender, EventArgs e)
        {
            DTE dte;
            dte = (DTE)GetService(typeof(DTE));
            string myString =
              "Name is " + dte.Name + "\rVersion is " + dte.Version;

            Windows windows = dte.Windows;
            Window window =
            (Window)windows.Item("{34E76E81-EE4A-11D0-AE2E-00A0C90FFFC3}");
            window.Visible = true;

            OutputWindow outputWindow = (OutputWindow)window.Object;
            OutputWindowPane outputWindowPane =
            outputWindow.OutputWindowPanes.Add("Test");
            outputWindowPane.Activate();
            outputWindowPane.OutputString(myString);
        }